docs/breaking-changes.md

```diff
@@ -98,7 +98,9 @@ is the origin that is checking for device permission.
 
 ## Planned Breaking API Changes (19.0)
 
-None
+### Removed: IA32 Linux binaries
+
+This is a result of Chromium 102.0.4999.0 dropping support for IA32 Linux.
```

Comment:
Probably want to link to the `Removed: IA32 Linux support` from here to highlight previous drop in support ?

Reply:
Sounds good, done!